python/ml4ir/applications/ranking/t_test.py

```diff
@@ -1,5 +1,324 @@
-from scipy import stats
 import numpy as np
+import scipy
```

Comment:
@mohazahran This should be usable for all `ml4ir` apps. Can you make it generic? I understand this was initially under `ranking/` but IMO that's not right too. You don't have to integrate it with Classification, but I think you should keep the option open for someone who wants to use this as an `import t_test` when using ml4ir to train models.

Reply:
ping.

Just move it to `ml4ir.base.stats` for now. We can refactor and add to ClassificationPipeline later.